age-quiz.py

```diff
@@ -14,6 +14,17 @@
 # so the flow of logic would output the first conditional 
 # and not need to output the next two.
 
+# Open the file in read mode
```

Comment:
Comments should mention that the 'drivers.txt' filename was changes to 'driver-info.txt' for clarification